teams/Aragon One/2019.md

```diff
@@ -19,7 +19,7 @@ The goal is to build an identity experience that is seamless and allows individu
 
 Focus areas for this initiative:
 
-- **Individual identity**: Allow individuals to create and manage their user profile, mapping their address to a human readable ENS name that can be used to interact with apps within the organization.
+- ✅ **Individual identity**: Allow individuals to create and manage their user profile, mapping their address to a human readable ENS name that can be used to interact with apps within the organization.
```

Comment:
I would argue this is not complete, but as of now we're shifting to supporting Autark/OWL's efforts on this.